lib/internal/util.js

```diff
@@ -356,6 +356,23 @@ function getConstructorOf(obj) {
   return null;
 }
 
+/**
+ * Get the current working directory while accounting for the possibility that it has been deleted.
+ * `process.cwd()` can fail if the parent directory is deleted while the process runs.
+ * @returns {string} The current working directory or the volume root if it cannot be determined.
+ */
+function getCwdSafe() {
+  let cwd;
+
+  try {
+    cwd = process.cwd() + '/';
+  } catch {
+    cwd = '/';
+  }
+
+  return cwd;
```

Comment:
so it works properly on windows
```suggestion
  let cwd = '';

  try {
    cwd = process.cwd();
  } catch {
    /**/
  }

  return cwd + path.sep;
```

Reply:
hm, i'm not sure - if it was only used in cases where the separator is normalized, it might be fine by accident. However, `process.cwd()` is a path, and paths (unlike URLs) use `path.sep` instead of `/`.

Reply:
Windows CI is passing. But maybe I could do something like this to prevent future problems?

```js
/**
 * Get the current working directory while accounting for the possibility that it has been deleted.
 * `process.cwd()` can fail if the parent directory is deleted while the process runs.
 * @returns {string} The current working directory or the volume root if it cannot be determined.
 */
function getCwdSafe() {
  let cwd;

  try {
    cwd = process.cwd() + path.sep;
  } catch {
    cwd = path.sep;
  }

  return cwd;
}
```

Reply:
sure, that's equivalent to my suggestion (except that my suggestion only adds the separator in one place, reducing the chance it'll be forgotten in the future)